v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerTest.java

```diff
@@ -80,4 +88,53 @@ public void testGetSourceTypeWithGcpCredentialsMissing() {
             .as(DataStreamToSpanner.Options.class);
     String result = DataStreamToSpanner.getSourceType(options);
   }
+
+  @Test
+  public void testConfigureSchemaOverrides_fileBased() {
+    DataStreamToSpanner.Options options = mock(DataStreamToSpanner.Options.class);
+    when(options.getSchemaOverridesFilePath())
+        .thenReturn(
+            Resources.getResource("DataStreamToSpannerFileOverridesIT/override.json").getPath());
+    when(options.getTableOverrides()).thenReturn("");
+    when(options.getColumnOverrides()).thenReturn("");
+
+    ISchemaOverridesParser parser = DataStreamToSpanner.configureSchemaOverrides(options);
```

Comment:
can you add checks to the actual expected values in the overrides ?

Reply:
done